src/patches/Node.js

```diff
@@ -307,7 +306,7 @@ export const NodePatches = utils.getOwnPropertyDescriptors({
       });
     }
     // if a slot is added, must render containing root.
-    if (this.localName === 'slot' || slotsAdded.length) {
+    if (slotsAdded.length) {
```

Comment:
This can be removed because `treeVisitor` visits the root node?

Reply:
It was incorrect. This says "the node we're inserting into is a slot" and in this case a slot was not added. What it was meant to trap was `node.localName` but this is not needed because the `treeVisitor` checks `node`.